int32 planner_id
geometry_msgs/Pose goal
float32 robot_size